### PR TITLE
Detailed forecast cleanup

### PR DESCRIPTION
**Pull request by lymanepp**

### CODE REVIEW

File: pynws/const.py [hunk 1]
```diff
@@ -60,3 +61,65 @@
     "blizzard": "Blizzard",
     "fog": "Fog/mist",
 }
+
+
+class Detail(StrEnum):
```
Comment (lymanepp) [inline pynws/const.py:66]: I like how this reads when referenced (e.g., Detail.TEMPERATURE). Maybe we can do the same for the SimpleForecast properties (e.g., Simple.TEMPERATURE). Thoughts?

Comment (MatthewFlamm) [reply pynws/const.py:66]: The only comment I have is whether we should structure/name this to indicate that it is for forecasts.  What happens when/if we add an StrEnum for the observations?

We could have `Detail`, `Simple`, `Observation`?

Comment (lymanepp) [reply pynws/const.py:66]: Either way is fine with me, but I have a slight preference for the terser versions (Detail, Simple and Observation).

**Comment by MatthewFlamm** [comment]
LGTM thanks!

**Comment by lymanepp** [comment]
Thanks for merging @MatthewFlamm! My plan is to eventually use latest version in a new HA component that will use the detailed forecast to calculate the comfort forecast for the next 24 hours. Something that is not possible with any other HA component that I've found so far. I'm not sure whether that would make any sense to be part of pynws or not.

**Comment by MatthewFlamm** [comment]
I'm not sure exactly what you are proposing would be part of pynws. I think any simple and widely useful extension of the data is in scope.  For example, I've converted wind speed ranges to an average in the simple forecasts.  Any HA component specific handling would be out of scope.

**Comment by MatthewFlamm** [comment]
Another thought is that I'm +1 on either replacing or augmenting the existing simple hourly forecast with the detailed forecast in the official ha integration.

**Comment by lymanepp** [comment]
> Another thought is that I'm +1 on either replacing or augmenting the existing simple hourly forecast with the detailed forecast in the official ha integration.

Maybe we should deprecate the existing simple hourly forecast and use the detailed forecast with some augmentation. The simple hourly forecast contains several values which are derived from the grid data--https://github.com/weather-gov/api/discussions/202

* isDaytime
* temperatureTrend
* icon
* shortForecast
* detailedForecast

Alternatively, we could derive those values directly and avoid a second API call. It might be too ambitious to reverse engineer the logic to always match.

Here's the hourly forecast raw format for reference.
```
{
	"number": 1,
	"name": "",
	"startTime": "2022-03-13T14:00:00-04:00",
	"endTime": "2022-03-13T15:00:00-04:00",
	"isDaytime": true,
	"temperature": 69,
	"temperatureUnit": "F",
	"temperatureTrend": null,
	"windSpeed": "14 mph",
	"windDirection": "NE",
	"icon": "https://api.weather.gov/icons/land/day/few?size=small",
	"shortForecast": "Sunny",
	"detailedForecast": "",
	"iconTime": "day",
	"iconWeather": [
		[
			"A few clouds",
			null
		]
	],
	"windBearing": 45.0,
	"windSpeedAvg": 14
}
```

**Pull request merged**